core/org.eclipse.cdt.core/parser/org/eclipse/cdt/internal/core/parser/scanner/DateMacro.java

```diff
@@ -36,7 +36,11 @@ private char[] createDate() {
 		DateFormatSymbols dfs = new DateFormatSymbols();
 		buffer.append(dfs.getShortMonths()[cal.get(Calendar.MONTH)]);
```

Comment:
I'm also a bit puzzled about this field as the compiler documentations appears to imply that the string is not translated, but the `java.text.DateFormatSymbols` is a way to retrieve the localized name of the month.

Maybe we should do as GCC does and just have an array of 3 letter English names of the months and pick from that?

Reply:
@Torbjorn-Svensson I agree that the `__DATE__` macro should provide a non-localized string according to the ANSI standard. So my suggestion would be to specify an appropriate locale in the `DateFormatSymbols` constructor:

`new DateFormatSymbols(Locale.ENGLISH).getShortMonths()`

Reply:
Thanks for the suggestion @jld01!